zerver/lib/user_groups.py

```diff
@@ -524,27 +523,58 @@ def get_group_setting_value_for_api(
     )
 
 
+@dataclass
+class UserGroupMembersDict:
```

Comment:
We already have class named `AnonymousSettingGroupDict` but this data structure is being used for named groups as well. Maybe we can just use this class name for all cases.

Reply:
Yeah, using this name across the board seems OK.